zsh-aws-vault.plugin.zsh

```diff
@@ -83,6 +92,9 @@ function _using_osx() {
 }
 
 function _find_browser() {
+  if [ -n "$AWS_VAULT_PL_BROWSER" ]; then
+    echo "$AWS_VAULT_PL_BROWSER"
+  fi
```

Comment:
potentially, _find_browser could cache the resolved browser name in this for later.

Reply:
I don't think that's a blocker for this PR.